src/main/java/com/google/sps/servlets/EventsServlet.java

```diff
@@ -81,18 +92,33 @@
     static final String START_TIME = "start";
     static final String END_TIME = "end";
     static final String USER_TIMEZONE = "timezone";
+    static final String CREATOR = "creator";
+    static final String EVENT_NUM = "EventNum";
+    static final String EVENT_NUM_VALUE = "value";
+    static final String EVENT_ID = "event_id";
+    static final String ID_TOKEN_PARAM = "id_token";
+
+    private static final String CLIENT_ID = "605480199600-e4uo1livbvl58cup3qtd1miqas7vspcu.apps.googleusercontent.com";
```

Comment:
What is this magic string? Is it unique for each client? Could update documentation on where you them from?